Resources/views/Comment/partial.comment.html.twig

```diff
@@ -0,0 +1,36 @@
+{% set commentId = parent|default ? parent.id : comment.id %}
+{% set replyId = parent|default ? comment.id : null %}
+
+<div class="media">
+    {% if comment.author.picture|default(comment.author.cover|default) %}
+        <div class="media-left">
+            <img class="media-object img-circle" src="{{ image(comment.author.picture|default(comment.author.cover)).cropResize(50) }}">
+        </div>
+    {% endif %}
+    <div class="media-body">
+        {% if comment.author|default %}
+            <h4 class="media-heading">{{ comment.author }}</h4>
+        {% endif %}
+        <p>
+            <small>
+                <span class="date">
+                    {{ comment.date|date('d-m-Y H:s') }}
+                </span>
+            </small>
+            -
+            <a class="delete-comment" href="
```

Comment:
The buttons don't look nice, no need to show them as comment, when it needs to be implemented someone has to see how they must look like.